azurerm/internal/services/postgres/postgresql_server_resource.go

```diff
@@ -609,6 +609,25 @@ func resourcePostgreSQLServerCreate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
+	// Issue tracking REST API update: https://github.com/Azure/azure-rest-api-specs/issues/14117
```

Comment:
random commit to trigger CI
```suggestion
	// Issue tracking the REST API update failure: https://github.com/Azure/azure-rest-api-specs/issues/14117
```